src/data/queries/vamcSystemVaPolice.ts

```diff
@@ -70,13 +70,36 @@ export const formatter: QueryFormatter<
   VamcSystemVaPolice
 > = ({ entity, menu }) => {
   const formattedMenu = buildSideNavDataFromMenu(entity.path.alias, menu)
+  // Similarly, this formats centralized content FAQs to match what our QA components are expecting
+  const buildFaqs = (faqs) => {
```

Comment:
From what I remember of Eli's original PR, there are going to be a lot of these little helper functions. How do you feel about extracting it into the top-level module scope to keep this function a little tighter? It looks like a pure function (of the flavor that requires no outside variables), so that should be a quick lift-and-shift.

Reply:
@cvalarida I have moved this. When you have a chance, let me know if this refactor makes sense.